src/posix/platform/configuration.cpp

```diff
@@ -43,6 +43,17 @@
 namespace ot {
 namespace Posix {
 
+namespace {
+#if OPENTHREAD_CONFIG_PLATFORM_POWER_CALIBRATION_ENABLE
+const char kKeyCalibratedPower[] = "calibrated_power";
+#endif
+const char kKeyTargetPower[]          = "target_power";
+const char kKeyRegionDomainMapping[]  = "region_domain_mapping";
+const char kKeySupportedChannelMask[] = "supported_channel_mask";
+const char kKeyPreferredChannelMask[] = "preferred_channel_mask";
+const char kCommaDelimiter[]          = ",";
+} // namespace
```

Comment:
Thinking more on this, would it be better to keep the constants as `private` definitions in the `class Configuration`  itself (since they are intended for use by this class only and would be namespaced under its name)? Adding them in anonymous `namespace` in `cpp` file restrict them to this file.